entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-set -u
+# Error handling
+set -euo pipefail
```

Comment:
Please explain what each of these options do as well as the 'pipefail' statement.

Reply:
done